src/pages/dashboard/components/transafer-graph/transfer-graph.component.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from "react";
+import { Paper, Box, Grid, CircularProgress } from "@mui/material";
+import { users } from "data/user";
+import Chart from "react-apexcharts";
+
+export const TransferGraph = () => {
+  const [transactions, setTransactions] = useState([
+    {} as {
```

Comment:
what's the advantage of using `as` instead of defining an interface and using it like this:
`useState<TransactionData[]>([])`

Reply:
I have to specify as empty object with a type by "as" even after the way you mentioned otherwise it gives me values possibly undefined in the code below. I have also made a types folder and moved the types there and used your method along with mine because it gives me the error I told you about if I only do it with yours.

Reply:
@Nosherwan-Akram please bring this up in our next call, we'll go over this warning. I think I know how to resolve it